src/sentry/api/endpoints/auth_index.py

```diff
@@ -70,6 +70,7 @@ def _verify_user_via_inputs(validator, request):
         # See if we have a u2f challenge/response
         if "challenge" in validator.validated_data and "response" in validator.validated_data:
             try:
+                metrics.incr("2fa.login_attempt", sample_rate=1.0, skip_internal=False)
```

Comment:
Would it make sense to put this on Line 96 instead, so it can be `2fa.success` instead?

Reply:
Also, a better name might be `auth.2fa.attempt` so in the external tool, all auth-related tags are prefixed and easy to find.